.github/workflows/publish-package.yml

```diff
@@ -19,20 +19,14 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0  # avoids shallow checkout as needed by setuptools-scm
-    - name: Set up Conda Environment
-      uses: conda-incubator/setup-miniconda@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
       with:
-        miniconda-version: "latest"
-        activate-environment: "pyscaffold"
-        environment-file: environment.yml
-        auto-update-conda: true
-        auto-activate-base: false
-    - name: Build and publish
+        python-version: 3.7
+    - name: Build Project and Publish
       env:
-        TWINE_USERNAME: "PyScaffold_Github"
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
         TWINE_PASSWORD: ${{ secrets.PYPI_PYSCAFFOLD_GITHUB }}
```

Comment:
Should we rather rename this to PYPI_PASSWORD? I mean otherwise, we are still giving away the username. In case we do, one would have to change that everywhere.

Reply:
Hi @FlorianWilhelm, I think the renaming is a good idea. I can not do it, so I decided to leave it the way it was 😝 (Github does not allow renaming a secret, only creating a new one).

This template was copied from `--custom-extension`. I like the idea of having it as vars there at least, so if a third-party creates an the extension in they can simply configure a different user/password without changing the file.

Reply:
Also, the reason why I went for a `secrets.PYPI_USERNAME` instead of a `env.PYPI_USERNAME`, despite the username not being a secret, is that it seems to me that Github still does not allow organisation-wide env vars, just secrets, and configuring the username repo by repo would be annoying...

Reply:
I created a secret `PYPI_PASSWORD` which is the same as `PYPI_PYSCAFFOLD_GITHUB` and removed the latter one. So we can change it in this PR to be more consistent. I will do it in the pyscaffoldext_dsproject extension.

Reply:
Thank you very much. It should be fine in master now ;)